projects/demo/src/modules/components/app-bar/index.html

```diff
@@ -7,9 +7,9 @@
         Component for the main app header
 
         <tui-doc-example
-            *ngFor="let example of examples; let index = index"
+            *ngFor="let example of examples; let index = index; let last = last"
             [component]="index + 1 | tuiComponent"
-            [content]="index + 1 | tuiExample: 'html,ts'"
+            [content]="index + 1 | tuiExample: (last ? 'html,ts,less' : 'html,ts')"
```

Comment:
Why we hide `less`-files for some examples ?

Should we reveal them ?
```suggestion
            [content]="index + 1 | tuiExample"
```

https://github.com/taiga-family/taiga-ui/blob/80fdfd0a1bd28f34d86b837c1f33c67ec9d7d139/projects/demo/src/utils/example.pipe.ts#L31